app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -254,12 +225,6 @@ def empty_file?(file)
         (file.respond_to?(:tempfile) && file.tempfile.size == 0) || (file.respond_to?(:size) && file.size == 0)
       end
 
-      # this is provided so that implementing application can override this behavior and map params to different attributes
```

Comment:
I don't really care about this change.  I do think we should endeavor to keep our PRs small and not change things that aren't related to the stated purpose of the PR.  I would have liked to have seen this change in a separate PR like "Reduce indirection in FileSetsController"